docs/introduction.md

```diff
@@ -9,15 +9,17 @@ slug: /introduction
 
 ![kwil-banner](../docs/kwil-banner.svg)
 
-## The permissionless SQL database for Web3-enabled applications
+## Web3 Native Decentralized Database Networks
 
-Kwil is the first permissionless database that enables structured data storage across a decentralized network. Kwil extends the functionality of traditional relational databases by providing a layer on-top of existing engines that enforce rules necessary for building secure, permissionless data stores.
+Kwil is a framework for deploying relational databases as standalone, byzantine fault tolerant networks. Kwil extends the functionality of traditional relational databases by providing a practical byanztine fault tolerant (pBFT) layer on top of existing engines, enforcing rules necessary for building secure, decentralized data stores.
```

Comment:
`for deploying` should be more general I think

Reply:
or `leveraging`, which would encompass deploying and querying

Reply:
yeah maybe `leveraging`